packages/ng/styles/components/cdk/_global.scss

```diff
@@ -10,11 +12,8 @@
 }
 
 .cdk-global-scrollblock {
-	&,
-	body {
-		touch-action: none;
-		overscroll-behavior: none;
-		-webkit-overflow-scrolling: auto;
-		overflow: hidden;
-	}
+	position: static !important;
+	width: auto !important;
+
+	@include overflow.block('!important');
```

Comment:
Width the property `overflow-block` beeing a thing, maybe this mixin name could bring confusion in the future ?

Reply:
Agree, and even without the confusion `overflow.block` doesn't seem crystal clear imho. 
What about something like `scroll.block` or `overflow.scrollblock` ?